src/Service/SharedManagementService.cs

```diff
@@ -409,6 +412,25 @@ public Task DisableGenerateSignInTokenEndpoint(string appId)
         return storage.DisableGenerateSignInTokenEndpoint();
     }
 
+    public async Task<IEnumerable<ConfiguredAuthenticatorResponse>> ListConfiguredAuthenticatorsAsync(ConfiguredAuthenticatorRequest request)
+    {
+        var storage = tenantFactory.Create();
+        var entities = await storage.GetAuthenticatorsAsync(request.IsAllowed);
+        return entities.Select(x => new ConfiguredAuthenticatorResponse(x.AaGuid, x.CreatedAt)).ToList();
+    }
+
+    public Task WhitelistAuthenticatorsAsync(WhitelistAuthenticatorsRequest request)
+    {
+        var storage = tenantFactory.Create();
+        return storage.WhitelistAuthenticatorsAsync(request.AaGuids);
+    }
+
+    public Task DelistAuthenticatorsAsync(DelistAuthenticatorsRequest request)
+    {
+        var storage = tenantFactory.Create();
+        return storage.DelistAuthenticatorsAsync(request.AaGuids);
+    }
```

Comment:
issue: if Authenticators are scoped "per tenant" and/or always modified in an app-context, they should not live in the SharedManagementService. SharedManagementService is for non-app/tenant specific scenarios.

sggestion: Since it seems like these Authenticators do not do any business-logic, it seems it does not warrant its own service but could instead live in `ApplicationService`?